internal/api/token.go

```diff
@@ -23,9 +23,9 @@ import (
 // AccessTokenClaims is a struct thats used for JWT claims
 type AccessTokenClaims struct {
 	jwt.RegisteredClaims
-	Email                         string                 `json:"email"`
-	Phone                         string                 `json:"phone"`
-	AppMetaData                   map[string]interface{} `json:"app_metadata"`
+	Email                         string                 `json:"email,omitempty"`
+	Phone                         string                 `json:"phone,omitempty"`
+	AppMetaData                   map[string]interface{} `json:"app_metadata,omitempty"`
 	UserMetaData                  map[string]interface{} `json:"user_metadata"`
```

Comment:
```suggestion
	UserMetaData                  map[string]interface{} `json:"user_metadata,omitempty"`
```

noticed this while testing by excluding `user_metadata` in `GOTRUE_JWT_ADDITIONAL_CLAIMS` and got the following payload in the JWT returned:
```json
{
  "iss": "https://projectref.supabase.co",
  "sub": "0c402da8-f1ac-4bf6-a39e-8f09078940af",
  "aud": "authenticated",
  "exp": 1837442173,
  "iat": 1737442173,
  "user_metadata": null,
  "role": "",
  "aal": "aal1",
  "session_id": "6cfb8463-249a-4a87-b3f8-de0986839220",
  "is_anonymous": false
}
```

Reply:
good catch!